Kitodo/src/main/java/org/kitodo/production/services/workflow/WorkflowControllerService.java

```diff
@@ -652,14 +647,12 @@ private boolean runXPathCondition(Process process, String xpath) throws IOExcept
         return ServiceManager.getProcessService().getNodeListFromMetadataFile(process, xpath).getLength() > 0;
     }
 
-    private void verifyTask(Task task) {
+    private void checkForAutomaticTask(Task task) {
```

Comment:
Maybe there is a better method naming possible like `perfomActionsForAutomaticTask` or something similar which is describing what this method is doing.

Reply:
renamed the method